client/pdo/client/scripts/CreateCLI.py

```diff
@@ -290,15 +307,25 @@ def Main(commands) :
     if options.ledger :
         config['Sawtooth']['LedgerURL'] = options.ledger
 
-    # set up the service configuration
+   # set up the service configuration
     if config.get('Service') is None :
         config['Service'] = {
+            'EnclaveServiceNames' : [],
             'EnclaveServiceURLs' : [],
-            'ProvisioningServiceURLs' : []
+            'ProvisioningServiceURLs' : [],
+            'EnclaveServiceDatabaseFile' : None
         }
+
+    if options.eservice_name:
+        use_eservice = True
+        config['Service']['EnclaveServiceNames'] = options.eservice_name
+    if options.eservice_db:
+        config['Service']['EnclaveServiceDatabaseFile'] = options.eservice_db
     if options.eservice_url :
+        use_eservice = True
         config['Service']['EnclaveServiceURLs'] = options.eservice_url
     if options.pservice_url :
+        use_pservice = True
         config['Service']['ProvisioningServiceURLs'] = options.pservice_url
```

Comment:
what are these flags for? for create, you don't have a choice. there must be eservices & pservices specified. so these are required.

Reply:
strange that i added this, will fix this. thank you